client/deployments/client.go

```diff
@@ -157,6 +191,87 @@ func listArtifact(a artifactData, detailLevel int) {
 	fmt.Println("--------------------------------------------------------------------------------")
 }
 
+func (c *Client) DirectUpload(
+	artifactPath, token, url string,
```

Comment:
Unused argument `token`. The signed URL is already authorized (embedded in the URL).
```suggestion
	artifactPath, url string,
```

Reply:
fixed.